app/http.php

```diff
@@ -985,10 +985,8 @@ function getStats(Table $statsHost, Table $statsContainers, Orchestration $orche
             }
         }
 
-        // TODO: @Meldiron Might cause stack overflow due to infinite recursion
         if ($recursive) {
-            \sleep(1);
-            getStats($statsHost, $statsContainers, $orchestration, $recursive);
+            Timer::after(1000, fn () => getStats($statsHost, $statsContainers, $orchestration, $recursive));
```

Comment:
PR looks good 👍  Just want to understand how this function works and how it solves the problem

Reply:
(wild guess how Swoole works)
Using timer::after creates a separate coroutine for future and instantly finishes current coroutine. In JavaScript world I would guess that these would translate to:

Old:
```js
function getStats() {
  getStats()
}
```

New:
```js
function getStats() {
  setTimeout(() => getStats(), 0);
}
```

Reply:
Alright sounds legit 👍